gobblin-runtime/src/main/java/gobblin/scheduler/PathAlterationListenerAdaptorForMonitor.java

```diff
@@ -186,6 +186,10 @@ private void rescheduleJob(Properties jobProps)
     boolean runOnce = Boolean.valueOf(jobProps.getProperty(ConfigurationKeys.JOB_RUN_ONCE_KEY, "false"));
     // Reschedule the job with the new job configuration
     jobScheduler.scheduleJob(jobProps, runOnce ? new RunOnceJobListener() : new EmailNotificationJobListener());
+    LOG.info("[JobScheduler]"
```

Comment:
This looks like a debug message. If so, please use `LOG.debug`, also, why is it in multiple lines?